### PR TITLE
Code sync blog1 to blog2

### PR DESCRIPTION
**Pull request by cwmeijer**
Mostly code cleaning and syncing. Please review.

### CODE REVIEW

File: js-webapp-async/newtonraphson.hpp [hunk 1]
```diff
@@ -1,15 +1,11 @@
...
-      public:
-         NewtonRaphson(double tolerancein);
-         double solve(double xin);
-      private:
-         double tolerance;
-  };
-}
+class NewtonRaphson {
+   public:
+      NewtonRaphson(double tolerancein);
```
Comment (sverhoeven) [inline js-webapp-async/newtonraphson.hpp:6]: The variables should be float instead of double. Also the var name changed.

**Review by sverhoeven — CHANGES_REQUESTED** [review]

File: js-webapp-async/newtonraphson.hpp (hunk 1 above)
Comment (sverhoeven) [reply js-webapp-async/newtonraphson.hpp:6]: Replace tolerancein with tolerance_in and xin with initial_guess

**Comment by sverhoeven** [comment]
Also algebra.cpp changed in first blog, please update aswell. See #33

File: js-webapp-async/newtonraphson.hpp (hunk 1 above)
Comment (jspaaks) [reply js-webapp-async/newtonraphson.hpp:6]: I fixed these in PR #75

**Comment by jspaaks** [comment]
> Also algebra.cpp changed in first blog, please update aswell. See #33

I fixed these in PR #75

**Pull request merged**